CAP/gap/Derivations.gi

```diff
@@ -43,7 +43,7 @@ function( name, target_op, used_op_names_with_multiples_and_category_getters, we
     fi;
     # =#
     
-    if NumberArgumentsFunction( category_filter ) = 0 or NumberArgumentsFunction( category_filter ) > 1 then
```

Comment:
This is deliberate: Functions with a variable number of arguments have a negative `NumberArgumentsFunction`, e.g.
```
gap> NumberArgumentsFunction( {x...} -> 1 );
-1
```
and those should work here too, although I would not encourage this, which is why it's not mentioned in the error message.

Reply:
Ah, thank you! Sorry for the inconvenience.